draft-ietf-quic-multipath.md

```diff
@@ -616,7 +592,36 @@ be controlled by the combination of one or several of the following:
    uses a series of consecutive sequence numbers without creating
    holes.
 
-### ACK Delay Considerations
+### NULL-CID-Loss-and-Congestion
+
+Implementations MUST implement per-path congestion control. Congestion
+control regulates the amount of data in flight and the pacing of packets
+using estimates of path characteristics. The estimates can use a variety
+of signals, such as packet acknowledgements, loss detection, or measurements
+of round trip time. These events MUST be attributed to the path over
+which the corresponding packets were sent. When sending to a NULL CID
```

Comment:
Actually, the beginning of this paragraph is not specific to the NULL CID case.

Reply:
Yes.

Reply:
Fixed.